test/integration/team-space-membership-integration.js

```diff
@@ -0,0 +1,42 @@
+export default function teamSpaceMembershipTests (t, space) {
+  t.test('Create, update and delete teamSpaceMembership', (t) => {
+    t.plan(3)
+    // create space membership for existing team
+    return space.createTeamSpaceMembership('5vllqmpyrhlgaz0xb2S90C', {
+      admin: false,
+      roles: [
+        {
+          sys: {
+            type: 'Link',
+            linkType: 'Role',
+            id: '6YFO0dKMUjeXB5OPoWnoNm' // role developer
+          }
+        }
+      ]
+    })
+      .then((response) => space.getTeamSpaceMembership(response.sys.id))
+      .then(teamMembership => {
+        teamMembership.roles = [
+          {
+            sys: {
+              type: 'Link',
+              linkType: 'Role',
+              id: '6YFc9mw7PREXh5FQ1lKSN6' // change role to author
+            }
+          }
+        ]
+        return teamMembership
```

Comment:
do we want to add a call to update the teamMembership after the role change?

Reply:
@vbeleuta good catch! fixed 🙂